.github/actions/storage-layout/action.yml

```diff
@@ -40,7 +40,7 @@ runs:
     - name: Compare layouts
       if: steps.reference.outcome == 'success' && github.event_name == 'pull_request'
       run: |
-        node scripts/checks/compare-layout.js --head ${{ inputs.layout }} --ref ${{ inputs.ref_layout }} >> $GITHUB_STEP_SUMMARY
+        node scripts/checks/compare-layout.js --head ${{ inputs.layout }} --ref ${{ inputs.ref_layout }}
```

Comment:
Its confusing to have to go to the summary to get the details.

It made sens to send the gasreport there, because its something we want to have available for review without having to care what step produced it. But here we only check it is there is an error.

- either there are no errors, and we don't care where its send (report is empty)
- nor there are errors, and we expect to see them on the test that failed, not somewhere else.

Also, in GITHUB_STEP_SUMMARY it doesn't look that good (formating)